opm/autodiff/StandardWell_impl.hpp

```diff
@@ -347,14 +347,21 @@ namespace Opm
             }
 
             // Using total mobilities
-            EvalWell total_mob_dense = mob_perfcells_dense[0];
+            EvalWell total_mob_dense = mob[0];
             for (int componentIdx = 1; componentIdx < num_components_; ++componentIdx) {
-                total_mob_dense += mob_perfcells_dense[componentIdx];
+                total_mob_dense += mob[componentIdx];
             }
 
             // injection perforations total volume rates
+            const double Tw = well_index_[perf];
             const EvalWell cqt_i = - Tw * (total_mob_dense * drawdown);
 
+            // surface volume fraction of fluids within wellbore
+            std::vector<EvalWell> cmix_s(num_components_, EvalWell{numWellEq + numEq});
```

Comment:
I think that `EvalWell{numWellEq + numEq}` will construct an Evaluation with value equal to `numWellEq + numEq`, rather than value zero. I assume this is to make the diff smaller to the dynamic version, but this is misleading, and should probably be changed back to zero-initialization.

Reply:
Yes. It is a mistake introduced during cleaning up. Nice catch.